src/functions/negate.js

```diff
@@ -3,13 +3,39 @@ import {wrap} from "../core/wrap";
 // Get the complement of a function.
 export const negate = wrap({
     name: "negate",
+    summary: "Get the logical negation of a predicate function.",
+    docs: process.env.NODE_ENV !== "development" ? undefined : {
+        expects: (`
+            The function expects a single predicate function as its input.
+        `),
+        returns: (`
+            The function builds and returns a function returning true for
+            which inputs producing a falsey value when passed to the
+            inputted predicate and returning false for all inputs producing
+            a truthy value when passed to that predicate.
+        `),
+        examples: [
+            "basicUsage"
+        ],
+        related: [
+            "allPass", "anyPass", "nonePass"
+        ],
+    },
```

Comment:
Other docs object attributes might include `warnings` which informs the user of cases where e.g. an infinite loop could occur, or exceptional memory usage or performance cost, or really anything that might come as a surprise to the uninformed user. `detail` for when the summary description is sufficient to give the basic idea but not to completely describe functionality.

I might move the `sequences` attribute in here too since the errors PR obviates it from a purely functional perspective, such that it will indicate all of the sequence types that could possibly be returned by the function, assuming it returns a sequence. (Tricky: How can you cleanly express something like "the type you get when you slice the input sequence"?)